frontend/src/containers/Lobby.js

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import "./Lobby.css";
+import openSocket from "socket.io-client";
+
+const socket = openSocket("http://localhost:3000");
```

Comment:
Instead of hard coding this, you may want to add this to a config somewhere. Eventually, when we start getting things hooked up, you don't want to have to keep changing this here to go back and forth.